C4SGWeb/src/app/user/account/user-account.component.ts

```diff
@@ -162,6 +164,26 @@ export class UserAccountComponent implements OnInit {
 
   }
 
+
+  openModal(user) {
+      this.modalActions.emit({action: "modal", params: ['open']});
+      this.selectedUser=user;
+
+    }
+
+    closeModal() {
+      this.modalActions.emit({action: "modal", params: ['close']});
+    }
+
+    deleteUser(user:User): void{
+      console.log(user.id);
+      this.userService.delete(user.id).subscribe(
+        error => console.log(error)
+      );
+
```

Comment:
1. @chaitanyas16 you don't have to *resubmit*, you can push more commits to this branch.

2. that's up to @Code4SocialGood , i don't know the specifics